### PR TITLE
feat/SOF-6720

### PR DESCRIPTION
**Pull request by pranabdas**
- cloning via HTTPS alternative to SSH
- describe alternative way to generate API tokens via platform web UI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,7 +69,7 @@ In order to run or edit the examples:
 
 1. Assert an existing Mat3ra.com account. Examples require an account to run. New users can register [here](https://platform.mat3ra.com/register) to obtain one.
 
-2. Open [settings](utils/settings.json) and adjust it to provide the API authentication parameters. See the [corresponding example](examples/system/get_authentication_params.ipynb) to learn how to obtain the authentication parameters.
+2. Open [settings](utils/settings.json) and adjust it to provide the API authentication parameters. See the [corresponding example](examples/system/get_authentication_params.ipynb) to learn how to obtain the authentication parameters. It is also possible to generate API token by login to [Mat3ra platform](https://platform.mat3ra.com/), navigating to account preferences, and click the Generate new token button under API Tokens. More details can be found [here](https://docs.mat3ra.com/accounts/ui/preferences/api/).
```
Comment (mrakitin) [inline README.md:72]:
```suggestion
2. Open [settings](utils/settings.json) and adjust it to provide the API authentication parameters. See the [corresponding example](examples/system/get_authentication_params.ipynb) to learn how to obtain the authentication parameters. It is also possible to generate an API token by logging in to [Mat3ra platform](https://platform.mat3ra.com/), navigating to the Account Preferences, and clicking the 'Generate new token' button under API Tokens. More details can be found [here](https://docs.mat3ra.com/accounts/ui/preferences/api/).
```

File: examples/system/get_authentication_params.ipynb [hunk 2]
```diff
@@ -17,7 +17,7 @@
    "source": [
     "# Overview\n",
     "\n",
-    "This example shows how to log in to Mat3ra RESTFul API via [Login](https://docs.mat3ra.com/api/API/post_login) endpoint and generate API authentication parameters."
+    "This example shows how to log in to Mat3ra RESTFul API via [Login](https://docs.mat3ra.com/api/API/post_login) endpoint and generate API authentication parameters. Note that it is also possible to generate API Token manually via platform web UI as described in the project [README](https://github.com/Exabyte-io/api-examples/tree/dev#usage)."
```
Comment (mrakitin) [inline examples/system/get_authentication_params.ipynb:20]:
```suggestion
    "This example shows how to log in to Mat3ra RESTFul API via [Login](https://docs.mat3ra.com/api/API/post_login) endpoint and generate API authentication parameters. Note that it is also possible to generate an API token manually via platform web UI as described in the project [README](https://github.com/Exabyte-io/api-examples/tree/dev#usage)."
```

**Review by mrakitin — COMMENTED** [review]
Good points, @pranabdas! I've suggested minor corrections below.

**Review by mrakitin — APPROVED** [review]
I tested the `getpass` addition via Colab, and it worked fine.

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**